integreat_cms/api_client_firebase/apps.py

```diff
@@ -0,0 +1,54 @@
+"""
+Configuration of PushNotificationSender API app
+"""
+import json
+import logging
+import requests
+
+
+
+from django.apps import apps, AppConfig
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
+
+
+class PushNotificationSenderApiConfig(AppConfig):
+    """
+    PushNotificationSender API config inheriting the django AppConfig
+    """
+
+    name = "integreat_cms.push_notification_sender_api"
+    api_available = False
```

Comment:
```suggestion
```
I don't think it makes sense to keep track of the API availability... it's an interesting info at server start so we notice potential problems in our server logs, but I would assume that even if the API is unavailable at the time of the check, it is probably a temporary downtime and we don't need to "store" that the api is down - just try it again when a push notification is sent, and hope that the server is up by then...

Reply:
Good issue. - we should talk, how solve this in total. Johannes will talk to you. Then we ll change it as negotiated. 
My suggest: put this or a similar function as integration-test to staging-tests. We could ignore it, if it fails < 3 times, something like this. (Checking the availability of the service does make sense in my opinion. Specially as it is legacy-code / legacy API)

Reply:
I removed the availability check for now because in its current form it doesn't work.